code/modules/mob/living/carbon/xenomorph/ai/xeno_ai.dm

```diff
@@ -192,7 +192,7 @@
 		return TRUE
 
 	var/turf/next_turf = current_path[current_path.len]
-	var/list/L = LinkBlocked(src, loc, next_turf, list(src, current_target), TRUE)
+	var/list/L = LinkBlocked(src, loc, next_turf, list(src), TRUE)
```

Comment:
why change the xeno one too?

Reply:
I did a little research on this one, and remembered we accidentially left this bug when cooking interactions stuff with morrow
The bug in question is - xenos should attack when meeting mob as an obstacle, but they don't because of target being a part of the ignore list, which on it's own does nothing except prevent us from doing things related to bumping into current_target